src/GraphicsState.php

```diff
@@ -26,7 +26,7 @@ class GraphicsState
     /**
      * @param Matrix|null $ctm
      */
-    public function __construct(Matrix $ctm = null)
+    public function __construct($ctm = null)
     {
         if ($ctm === null) {
```

Comment:
Can you check for `(!$ctm instanceof Matrix)` here then?

Reply:
I added an explicit check here. Minimal semantic difference: before, we had a `TypeError` for invalid argument types, now it's an `InvalidArgumentException`. Shouldn't make much difference.